### DISCUSSION TITLE
Have the audio issues been resolved?

### DISCUSSION THREAD

**Discussion by norogoth** [pop-os/pop#2479] [Q&A]
I am on 20.04 because earlier when I upgraded my sound when out and I got "dummy output" when adjusting volume whether I had headphones in or not. When I used the fixes found in other thread like here - https://github.com/pop-os/pop/issues/1395 I was not able to resolve the issue and I just had to refresh my installation. Now I am on a less new version but I would like to be on the latest stable version.

**Comment by jackpot51** [comment]
You could try booting up the 22.04 ISO to see if it works there, prior to upgrading.

**Comment by norogoth** [comment]
Oh. I don't have the ISO as of yet but I guess you're right; I can just download it and boot to it on usb to see. OK thanks.
